addons/website_slides/models/slide_slide.py

```diff
@@ -401,29 +503,29 @@ def create(self, values):
             # 'website_published' is handled by mixin
             values['date_published'] = False
 
-        if values.get('slide_category') == 'infographic' and not values.get('image_1920'):
-            values['image_1920'] = values['binary_content']
         if values.get('is_category'):
             values['is_preview'] = True
             values['is_published'] = True
         if values.get('is_published') and not values.get('date_published'):
             values['date_published'] = datetime.datetime.now()
-        if values.get('url') and not values.get('document_id'):
-            doc_data = self._parse_document_url(values['url']).get('values', dict())
-            for key, value in doc_data.items():
-                values.setdefault(key, value)
 
         slide = super(Slide, self).create(values)
 
+        # avoid fetching external metadata when installing the module (i.e. for demo data)
+        # we also support a context key if you don't want to fetch the metadata when creating a slide
+        if any(values.get(url_param) for url_param in ['url', 'video_url', 'document_google_url', 'image_google_url']) \
```

Comment:
While we are doing "onchange-like stuff because of external data", shoudln't we call ``_on_change_document_binary_content`` if not ``completion_time`` is not given in creation values ? That way our create is complete.